rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/dto/PermissionForm.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive_grid_cloud_portal.common.dto;
+
+import java.util.List;
+
+
+/**
+ * A class that represent a mean to get jobIds and methods through the rest api.
+ * The jobIds list represents the list of job ids to check the permissions for
+ * The methods list represents the list of methods to check whether each jobId has permission
+ */
+public class PermissionForm {
```

Comment:
It is missing a constructor and annotations to specify how to serialize/deserialize the object, either json or xml

Reply:
I think rest easy uses the no arg constructor and getter/setters to build the object